qemu/qemu.go

```diff
@@ -42,6 +42,9 @@ type Config struct {
 	// Ctx is not used at the moment.
 	Ctx context.Context
 
+	// Name is the qemu guest name
+	Name string
+	
```

Comment:
Line 47 was showing a tab.  Either you found it or on some other edit/save goftmt removed it?  It's not there in what merged.